lib/elixir/lib/macro/env.ex

```diff
@@ -64,13 +64,13 @@ defmodule Macro.Env do
   @type macros :: [{module, [name_arity]}]
   @type context_modules :: [module]
   @type lexical_tracker :: pid | nil
-  @type var :: {atom, atom | term}
+  @type variable :: {atom, atom | term}
```

Comment:
This is also a breaking change, but the test suite won't pass even if we turn the error into a warning 😕